js/src/personal/amanda/Landing.page.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Center, Title } from '@mantine/core'
+import { useViewportSize } from '@mantine/hooks'
+import { MouseEventHandler } from 'react'
+import styles from './Amanda.module.css'
+
+export function LandingPage({ onClick }: { onClick: MouseEventHandler }) {
+  const { height, width } = useViewportSize()
+  return (
+    <>
+      <div className={styles.landingPage}>
+        <Center
+          w={width}
+          h={height / 2}
```

Comment:
This resizes when you resize the window right? I would expect useViewportSize() to handle that.